_sass/minima/skins/solarized.scss

```diff
@@ -5,8 +5,10 @@
 // Created by Sander Voerman <mailto:sander@savoerman.nl> using the Solarized
 // color scheme by Ethan Schoonover <https://ethanschoonover.com/solarized>.
 
-// This style sheet implements two options for the minima.skin setting:
-// "solarized" for light mode and "solarized-dark" for dark mode.
+// This style sheet implements three options for the minima.skin setting:
+// "solarized-light" for light mode, "solarized-dark" for dark mode, and
+// "solarized" for light or dark mode depending on user preference.
```

Comment:
IMO, this info should be exposed on the README as well.

Reply:
Like so? 5780ec40f2886d9593fc6f53dd2cca5fe2c8052e

Reply:
Yes. But reading the table from a novice-user's POV, its not clear how exactly is the `user preference` set.
Perhaps, a link to additional info (for a lay-user) should clear things.

Reply:
:+1:

Reply:
OK, it now mentions that this is done by operating system or browser settings, and references the explanation on MDN (or would that be too technical?)

Reply:
> (or would that be too technical?)

I won't let that block this PR.
Lets defer addressing that, if needed, for a future PR.